platform/pom.xml

```diff
@@ -270,6 +271,11 @@
                 <version>${tika.version}</version>
             </dependency>
             <dependency>
+                <groupId>org.codice.thirdparty</groupId>
+                <artifactId>tika-bundle</artifactId>
+                <version>${tika.thirdparty.bundle.version}</version>
+            </dependency>
```

Comment:
We went back and forth on this, but that's how it's being done throughout catalog and I couldn't get it to work any other way